src/Exception/ParameterNotAllowedException.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\PageBundle\Exception;
+
+class ParameterNotAllowedException extends SonataPageException
```

Comment:
We don't need this

Reply:
Why not? how do you group the exceptions? fro exaple

if I want to catch all exceptions from SonataPage like

```php
cache(SonataPageException $exception) ...
```

Reply:
Ok I got it :)